utils/build/docker/python/install_ddtrace.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eu
 
+# Install Rust toolchain
+curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable -y
```

Comment:
nit: maybe would be safer to cache  the script in the repository? Just in case rustup.rs is compromised, trough this a docker image so consider this a non blocking nit.

Reply:
not sure, this is similar to what we do in other places... but... 🤷🏻